mlflow/server/js/package.json

```diff
@@ -81,13 +83,10 @@
       "lcov"
     ]
   },
-  "proxy": {
```

Comment:
Moved to `src/setupProxy.js` as part of migration to create-react-app 2.0 ([see create-react-app docs](https://facebook.github.io/create-react-app/docs/proxying-api-requests-in-development#configuring-the-proxy-manually))